scripts/ft-benchmark-data-sender.py

```diff
@@ -89,24 +85,33 @@ def get_commit() -> tuple[str, datetime]:
 
 
 def commit_to_db(data: dict) -> None:
+    print(data)
     chdir(os.path.expanduser("~/nearcore/benchmarks/continous/db/tool"))
-    with tempfile.NamedTemporaryFile() as fp:
+    with tempfile.NamedTemporaryFile(mode="w", encoding='utf-8') as fp:
         json.dump(data, fp)
-        fp.close()
+        fp.flush()
+        os.fsync(fp.fileno()) 
         subprocess.run(f"cargo run -p cli -- insert-ft-transfer {fp.name}",
                        shell=True)
+        fp.close()
 
 
 # TODO: send signal to this process if ft-benchmark.sh decided to switch neard to another commit.
 # add handling of this signal to this script
 if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Process some integers.")
+    parser.add_argument('--duration', type=int, required=True, help='Duration of experiment in seconds')
+    DURATION = parser.parse_args().duration / 3600
+
     state_size = (int(
         subprocess.check_output(["du", "-s", "~/.near/localnet/node0/data"],
                                 stderr=subprocess.PIPE,
                                 shell=True).decode("utf-8").split()[0]) * 1024)
     processed_transactions = []
     time_begin = datetime.now()
     while True:
+        print(f"Data sender loop. Time elapsed: {(datetime.now() - time_begin).seconds} seconds")
```

Comment:
Let's use a `logging` module for this. It will automatically include the time in the log tine.

Reply:
Probably time from start of this loop is more informative than just time. But in general I added this just to see if this script started